modules/consul-cluster/main.tf

```diff
@@ -29,6 +29,12 @@ resource "aws_autoscaling_group" "autoscaling_group" {
     ],
     local.tags_asg_format,
   )
+
```

Comment:
Why do we need this?

Reply:
Because the ALB is associated using a separate, standalone resource.